### DISCUSSION TITLE
7.0.0-alpha.2 - Unexpected Exception: Unable to read data from the transport connection: Connection reset by peer

### DISCUSSION THREAD

**Discussion by plewam** [rabbitmq/rabbitmq-dotnet-client#1463] [Q&A]
I am currently testing the 7.0.0-alpha.2 release. I have set the `AutomaticRecoveryEnabled` to `true`. During the consume of messages from my remote server my VM somehow had it's network connection interrupted for a short period of time. I use all the new `async` methods.

During this short outage I receive these events from the library. Please not that these are my own logging texts.

1. (Channel) ChannelShutdown
2. (Consumer) ConsumerCancelled
3. (Consumer) Shutdown

`RabbitMQ model was shutdown. ReplyText was <"Unexpected Exception: Unable to read data from the transport connection: Connection reset by peer.">`

`RabbitMQ consumer was cancelled. Consumer tag <"71cd2fcf-0f86-477b-86d6-c9bf0518b51e">`

`RabbitMQ consumer was shut down. ReplyText <"Unexpected Exception: Unable to read data from the transport connection: Connection reset by peer.">. Initiator <Library>`

However I would expect the channel and consumer to automatically recover. Can somebody maybe give me some tips? Or is this issue already known?

**Comment by lukebakken** [comment]
@plewam this seems like a legitimate issue. Please continue discussion here: https://github.com/rabbitmq/rabbitmq-dotnet-client/issues/1464
